config/routes.rb

```diff
@@ -2,11 +2,13 @@
 
 Rails.application.routes.draw do
   root 'main#home'
+  get '/home', to: 'main#homeregistered', as: 'home'
+  get '/rules', to: 'main#rules', as: 'rules'
+  get '/welcome', to: 'main#welcome', as: 'welcome'
   devise_for :users,
-             controllers: { omniauth_callbacks: 'users/omniauth_callbacks' },
-             skip:        [:sessions, :registrations]
+             controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.